Program.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+
+namespace SteveAdventure
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //Introduction 
+            Console.WriteLine("Welcome to INTERGALACTIC TINDER!!!:");
+            Console.WriteLine("You are an alien scientist visiting earth to experience a date.");
+            Console.WriteLine("You see a strong brown-eyed human with short hair, and a lovely green eyed human with long red hair.");
+            Console.WriteLine("Which do you choose to approach? Type Brown or Green:");
+
+            string response = Console.ReadLine();
+
+            if (response == "Brown")
+            {// BROWN EYES --> HUNGRY (seafood) or FUN (football)
+                Console.WriteLine("This human makes you feel funny inside. They invite you on a date, asking if you are Hungry, or wish to view Fun human entertainment.");
+                Console.WriteLine("Type \"Hungry\" or \"Fun\":");
+
+                string responseB = Console.ReadLine();
+
+                if (response == "Hungry")
```

Comment:
This doesn't look like it will work as intended.  `response` was created on line 15 and still has the same value ("Brown", at this point).  Consider revising this if-statement to use `responseB` instead.